src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -716,6 +716,27 @@ export class StudyViewPageStore {
         }
     });
 
+    readonly getDataForClinicalDataTab = remoteData({
+        await: () => [this.clinicalAttributes, this.selectedSamples],
```

Comment:
are we just showing selected/filtered samples clinical data?

Reply:
Yup, JJ asked before to synchronize the tab with the current selection. So I added it in.